.github/ISSUE_TEMPLATE/questions.yml

```
@@ -0,0 +1,11 @@
+name: Change Request
+description: Suggest an idea.
```

Copy pasta from request.yml?
```suggestion
name: Question
description: Ask a question.
```